UPGRADE-4.4.md

```diff
@@ -108,9 +108,33 @@ HttpFoundation
 HttpKernel
 ----------
 
- * Implementing the `BundleInterface` without implementing the `getPublicDir()` method is deprecated.
-   This method will be added to the interface in 5.0.
  * The `DebugHandlersListener` class has been marked as `final`
+ * Added new Bundle directory convention consistent with standard skeletons:
+
+    ```
+    └── AcmeBundle
+        ├── config/
+        ├── public/
+        ├── src/
+        │   └── AcmeBundle.php
+        ├── templates/
+        └── translations/
+    ```
+
+   To make this work properly, it is necessary to change the root path of the bundle:
+
+    ```php
+    class AcmeBundle extends Bundle
+    {
+        public function getPath()
+        {
+            return \dirname(__DIR__);
+        }
+    }
+    ```
+
+   As bundles must be compatible with many Symfony versions we are not deprecating
```

Comment:
`As many bundles must be compatible with a range of Symfony versions,  the current directory convention is not deprecated yet, but it will be in the future.`

maybe duplicate in 5.0 file so that ppl can read about it with new 5.0 projects too

Reply:
done.